test/unittest/dds/subscriber/DataReaderTests.cpp

```diff
@@ -2571,6 +2571,45 @@ TEST_F(DataReaderTests, delete_contained_entities)
     ASSERT_EQ(data_reader->delete_contained_entities(), ReturnCode_t::RETCODE_OK);
 }
 
+/*
+ * This test checks the allocation consistency when using instances. If the number of instances is infinite,
+   and the max_samples is not set, the endpoint creation should fail. The following methods are checked:
```

Comment:
```suggestion
   and the max_samples is not, the endpoint creation should fail. The following methods are checked:
```

Reply:
Change implemented on commit:
e320216c5 Refs #14958: Adrian's suggestions: Change in comment description of the related tests to be more exact